docs/en/connector-v2/sink/Hive.md

```diff
@@ -28,78 +33,175 @@ By default, we use 2PC commit to ensure `exactly-once`
+| hive_site_path                | string  | no       | -                            | The path of `hive-site.xml`, used to authentication hive metastore                                                                                                                                                                                                          |
+| hive.hadoop.conf              | Map     | no       | -                            | Properties in hadoop conf('core-site.xml', 'hdfs-site.xml', 'hive-site.xml')                                                                                                                                                                                                |
+| hive.hadoop.conf-path         | string  | no       | -                            | The specified loading path for the 'core-site.xml', 'hdfs-site.xml', 'hive-site.xml' files                                                                                                                                                                                  |
+| krb5_path                     | string  | no       | /etc/krb5.conf               | The path of `krb5.conf`, used to authentication kerberos                                                                                                                                                                                                                    |
+| kerberos_principal            | string  | no       | -                            | The principal of kerberos                                                                                                                                                                                                                                                   |
+| kerberos_keytab_path          | string  | no       | -                            | The keytab path of kerberos                                                                                                                                                                                                                                                 |
+| abort_drop_partition_metadata | boolean | no       | true                         | Flag to decide whether to drop partition metadata from Hive Metastore during an abort operation. Note: this only affects the metadata in the metastore, the data in the partition will always be deleted(data generated during the synchronization process).                |
+| common-options                |         | no       | -                            | Sink plugin common parameters, please refer to [Sink Common Options](common-options.md) for details                                                                                                                                                                         |
+
+### schema_save_mode[Enum]
```

Comment:
Added to options

Reply:
it is already in options, just add more explanation